plugins/cairo-lang-macro/src/lib.rs

```diff
@@ -66,18 +69,59 @@ pub enum ProcMacroResult {
 }
 
 #[derive(Debug, Default, Clone)]
-pub struct TokenStream(String);
+pub struct TokenStream {
+    value: String,
+    metadata: TokenStreamMetadata,
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct TokenStreamMetadata {
+    original_file_path: Option<String>,
```

Comment:
Do we need file path if we have fileid ?

We wanted to include paths in spans anyway, and in this context both fields provide the same value

Reply:
Discussed offline